packages/tools/kadena-cli/src/utils/globalOptions.ts

```diff
@@ -195,6 +196,64 @@ export const globalOptions = {
       'Version of the kadena/devnet Docker image to use (e.g. "latest")',
     ),
   }),
+  //Simulate
```

Comment:
please take a look at this comment, since i have the same comment here for this code: https://github.com/kadena-community/kadena.js/pull/1476/files#r1453533628

Reply:
Fixed